data/lib/src/datasource_impl/shared_space_document_datasource_impl.dart

```diff
@@ -93,4 +103,30 @@ class SharedSpaceDocumentDataSourceImpl implements SharedSpaceDocumentDataSource
 
     return FileUploadState(mergedStream, UploadTaskId(taskId));
   }
+
+  @override
+  Future<List<WorkGroupNode>> getAllChildNodes(
+    SharedSpaceId sharedSpaceId, [WorkGroupNodeId parentNodeId]) {
+    return Future.sync(() async {
+      return (await _linShareHttpClient.getWorkGroupChildNodes(sharedSpaceId, parentNodeId))
+          .map<WorkGroupNode>((workgroupNode) {
+            if (workgroupNode is WorkGroupDocumentDto) return workgroupNode.toWorkGroupDocument();
+
+            if (workgroupNode is WorkGroupNodeFolderDto) return workgroupNode.toWorkGroupFolder();
+
+            developer.log('Cannot cast workgroup node = $workgroupNode');
```

Comment:
are you sure this log is not visible in `release`

Reply:
I think we should create a wrapper class for logging. smthg like `LinshareLogger`

Reply:
What is your idea? @manhduc-nguyen

Reply:
> are you sure this log is not visible in release?

Can you confirm that?

Reply:
Not found the docs say detail about `developer.log` '_' Investigate later after task add log :p

Reply:
please remove this log